Orso.Arpa.Domain/Entities/SphereOfActivityConcert.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+namespace Orso.Arpa.Domain.Entities
+{
+    public class SphereOfActivityConcert : BaseEntity
+    {
+        public SphereOfActivityConcert(Guid? id, MusicianProfile musicianProfile, Venue venue) : base(id)
```

Comment:
Wording: _Sphere of activity_ sounds spacy to me. I'd rather choose _area of activity_ of _activity area_